### PR TITLE
247: check kyc

### PR DESCRIPTION
**Pull request by Vardan-Alaverdyan**
Please register webhook from https://dashboard.onfido.com/api/webhook_management.
Now we need to  subscribe to `check.completed` event only.
Copy security token and add a new env called `ONFIDO_WEBHOOK_SECRET`.
Now one is registered for sandbox only.

Also added functionality for `check` in the `signups.js`. Uncomment `do_applicant_checks()` function and set a proper interval for it. (`CHECKS_INTERVAL` variable). 
Also please check the README.md

### CODE REVIEW

File: src/signups.js [hunk 1]
```diff
@@ -154,46 +206,60 @@ function send_new_signup_mail(user) {
...
+            }
+        } catch (error) {
+            console.error(error);
+            process.exit(1);
+        }
+    }, CHECKS_INTERVAL);
 }
 
 do_work();
+// do_applicant_checks();
```
Comment (nkostoulas) [inline src/signups.js]: ?

File: src/env.js [hunk 2]
```diff
@@ -67,5 +67,10 @@ module.exports = {
             from_name: env.MS_SMTP_FROM_NAME,
             from_address: env.MS_SMTP_FROM_ADDRESS
         }
+    },
+    onfido: {
+        webhook: {
+            secret: env.ONFIDO_WEBHOOK_SECRET
```
Comment (nkostoulas) [inline src/env.js]: All secrets should have the option to be read from a file as well.

File: src/signups.js [hunk 3]
```diff
@@ -9,33 +9,62 @@ const models = require('../src/models/models');
 
 mongoose.set('useFindAndModify', false);
 mongoose.set('useUnifiedTopology', true);
+mongoose.set('debug', false);
 
+const ONFIDO_REQUEST_HEADERS = {
```
Comment (nkostoulas) [inline src/signups.js]: You have this in two places.

Comment (Vardan-Alaverdyan) [reply src/signups.js]: yes

File: src/env.js (hunk 2 above)
Comment (nkostoulas) [reply src/env.js]: Same as other envs. Can you add that?

**Pull request merged**